wire-ios/Wire-iOS/Sources/UserInterface/Conversation Options/ConversationGuestOptionsViewModel.swift

```diff
@@ -232,6 +235,25 @@ final class ConversationGuestOptionsViewModel {
         }
     }
 
+    /// Starts the Guest Link Creation Flow
+    /// - Parameter view: the source view which triggers create action
+    func startGuestLinkCreationFlow(from view: UIView? = nil) {
+        guard let apiVersion = BackendInfo.apiVersion else { return }
+        if apiVersion >= .v4 {
```

Comment:
I would move this condition to the separate property. Something like:
isGuestLinkWithPasswordAvailable or isSecureGuestLink or ....
```
var isGuestLinkWithPasswordAvailable: Bool {
guard let apiVersion = BackendInfo.apiVersion else { return false }

return apiVersion >= .v4
}
```

Reply:
Done!